pkg/utils/utils.go

```diff
@@ -27,3 +34,16 @@ func GetEnvOrDefault(key, def string) string {
 	}
 	return val
 }
+
+func RecordEvent(ctx context.Context, recorder events.EventRecorder, object, relatedObject runtime.Object, name events.EventName, controller string) {
```

Comment:
I don't think this util method is useful. All it does is pass the parameters along to another function. If the purpose is to log the error, it is already done in the `RecordEvent` function and you can ignore the error here

Reply:
Agreed on the error logging part but, it can long lines of code for raising an event (preparing the event object) and instead a single line right ?

Reply:
@kmjayadeep How much time does it take ?
So that we can take a decision to fix now or have tech debt to fix latter